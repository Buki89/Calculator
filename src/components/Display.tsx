import React from "react";
import { Box, Expression, Result } from "../styles/display";

interface Props {
  expression: string;
  result: string;
}

const Display = (props: Props) => {
  const { expression, result } = props;
  return (
    <Box>
      <Expression>{expression.replace(".", ",")}</Expression>
      <Result>{result.replace(".", ",")}</Result>
    </Box>
  );
};

export default Display;
