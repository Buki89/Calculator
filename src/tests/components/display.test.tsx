import React from "react";
import renderer from "react-test-renderer";
import Display from "../../components/Display";

it("renders correctly when there are no items", () => {
  const tree = renderer
    .create(<Display result={""} expression={""} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
