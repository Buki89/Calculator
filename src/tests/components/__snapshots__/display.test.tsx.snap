// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="sc-dlnjPT bwAvQY"
>
  <div
    className="sc-gtssRu ljBYbT"
  >
    
  </div>
  <div
    className="sc-bdnylx bbYWlV"
    textLength={0}
  >
    
  </div>
</div>
`;
